.github/workflows/angular.yml

```diff
@@ -88,71 +88,71 @@ jobs:
                 include:
                     - app-type: ngx-default
                       entity: sql
-                      profile: prod
+                      environment: prod
                       war: 0
                       e2e: 1
                       testcontainers: 1
                     - app-type: ngx-mysql-es-noi18n-mapsid
                       entity: sqlfull
-                      profile: prod
+                      environment: prod
                       war: 0
                       e2e: 1
                       testcontainers: 0
                     - app-type: ngx-mariadb-oauth2-infinispan
                       entity: sqlfull
-                      profile: prod
+                      environment: prod
                       war: 0
                       e2e: 1
                       testcontainers: 0
                     - app-type: ngx-mongodb-kafka-cucumber
                       entity: mongodb
-                      profile: dev,webpack
```

Comment:
simple question: does it work without webpack profile ? The last time, it was not possible, what's the trick?

Reply:
@pascalgrimaud I believe this has been added to package.json file directly

Reply:
Added to scripts.